pages/common/cpio.md

```diff
@@ -0,0 +1,16 @@
+# cpio
+
+> Copies files in and out of archives.
+> Supports following archive formats: binary, old ASCII, new ASCII, crc, HPUX binary, HPUX old ASCII, old tar, and POSIX.1 tar.
+
+- Take a list of file names from standard input and copy them [o]ut to an archive in binary format:
+
+`echo "{{file1}} {{file2}} {{file3}}" | cpio -o > {{archive.cpio}}`
+
+- Copy [o]ut all files and folders in a directory to an archive, in [v]erbose mode:
+
+`find {{path/to/directory}} | cpio -ov > {{archive.cpio}}`
+
+- Copy [i]n all files from an archive, generating [d]irectories where needed, in [v]erbose mode:
```

Comment:
instead of "copy in files from an archive", the natural form is as it appears in the description: "copy files into an archive". Of course, these two sentences have the opposite meaning. Are you sure we're not reading the command options with their meanings reversed?